include/tvm/ir/type.h

```diff
@@ -175,8 +182,9 @@ class PointerType : public Type {
   /*!
    * \brief Constructor
    * \param element_type The type of the element which the pointer points to.
+   * \param storage_scope The storage scope into which the pointer addresses
    */
-  TVM_DLL explicit PointerType(Type element_type);
+  TVM_DLL explicit PointerType(Type element_type, String storage_scope = "global");
```

Comment:
To be consistent with the rest of the code base, let us still allow empty string, which indicates that it is global scope. We do not need to print out the scope of global ptr to simplify the readability of the Ptr type.